docs/Examples2/Clip.example.purs

```diff
@@ -1,24 +1,33 @@
 module Lumi.Components2.Examples.Clip where
 
 import Prelude
+
+import Lumi.Components (($$$))
 import Lumi.Components.Example (example)
 import Lumi.Components.Spacing (Space(..), vspace)
-import Lumi.Components.Text (body_, p_)
+import Lumi.Components2.Text as T
 import Lumi.Components2.Box (box)
 import Lumi.Components2.Clip (clip)
+import Lumi.Styles as S
 import React.Basic.Classic (JSX)
 
 docs :: JSX
 docs =
   box
     _
       { content =
-        [ p_ "The Clip component wraps the provided content with a grey border and a \"Copy\" button, which copies the text content into the system clipboard."
-        , p_ "If clipboard access is not allowed or not supported the text will be left highlighted, allowing the user to press ctrl+c manually. Only the plain text content is copied, not the HTML."
+        [ T.paragraph_ $$$ "The Clip component wraps the provided content with a grey border and a \"Copy\" button, which copies the text content into the system clipboard."
+        , T.paragraph_ $$$ "If clipboard access is not allowed or not supported the text will be left highlighted, allowing the user to press ctrl+c manually. Only the plain text content is copied, not the HTML."
         , vspace S24
         , example
             $ clip
-            $ _ { content = [ body_ "someone@email.com" ] }
-        , p_ "The Clip behavior is also available as a React hook."
+            $ _ { content =
+                    [ T.paragraph
+                      $ S.style_ (S.css { marginBottom: S.int 0 })
+                      $ T.truncate
+                      $$$ [ T.text $$$ "someone@email.com" ]
```

Comment:
Does truncated text impact the copy logic? I'd expect it not to since the string value in the html is still the full email, but in some browsers it's selecting text by programmatically highlighting the text, which might pull from the rendered text instead of the un-styled value.

Reply:
Tested it on Chrome and the copy logic remains the same; including when copying from the rendered text. But I will do some further browser testing to confirm.

Reply:
Tested on Chrome, Firefox, and Safari. All 3 browsers when using the copy action or copying from the highlighted rendered text (when truncated) work properly (uses the full string value).